src/client/sandbox/node/node-list-wrapper.js

```diff
@@ -0,0 +1,85 @@
+export default class NodeListWrapper {
+    constructor (nodeList, filterListFn, filterItemFn) {
+        Object.defineProperty(this, 'item', {
+            enumerable:   true,
+            configurable: false,
```

Comment:
Default value for `configurable`, `emumerable`, `writable` properties is `false`.
So, you can just remove `configurable: false`, `enumerable:   false`;